src/test/features/claim/routes/claimant-address.ts

```diff
@@ -52,7 +52,7 @@ describe('Claim issue: claimant address page', () => {
 
     it('should return 500 and render error page when form is valid and cannot save draft', async () => {
       idamServiceMock.resolveRetrieveUserFor('1', ...roles)
-      draftStoreServiceMock.rejectSave(1000, 'HTTP error')
+      draftStoreServiceMock.rejectSave(100, 'HTTP error')
```

Comment:
what's this for?

Reply:
retrieve claim was using 100 as default claim id, however update was against id 1000, so test was failing.